rgw/v2/lib/resource_op.py

```diff
@@ -126,6 +142,8 @@ def read(self):
         self.max_objects_per_shard = self.doc['config'].get('max_objects_per_shard')
         self.max_objects = None
         self.user_count = self.doc['config'].get('user_count')
+        self.user_create = self.doc['config'].get('user_create')
```

Comment:
`self.user_create = self.doc['config'].get('user_create')` 
can be written as 
`self.user_create = self.doc['config'].get('user_create', False)`
with this change, you dont have to make any changes in any yamls.